### ISSUE TITLE
Is it normal to set reqires_grad=False for D before update G?

### ISSUE THREAD

**Issue by yoon28** [AlexiaJM/RelativisticGAN#22]
Hi!
First, thank you for sharing the code. I appreciate it.
I have a question.
When you update the G, the gradient of D was not computed?
Is there a specific reason you set `requires_grad=False` for D before update the G?
The code is applied to all types of GAN including original GAN where the original implementation does not.

**Comment by AlexiaJM** [comment]
It means that you don't compute the gradient with respect to the D parameters, but the gradient is still calculated on the input and then the generator parameters. It's a tiny (negligible) speed-up.
